app/src/main/java/org/grapheneos/pdfviewer/fragment/DocumentPropertiesFragment.java

```diff
@@ -14,37 +15,35 @@
 import org.grapheneos.pdfviewer.R;
 
 public class DocumentPropertiesFragment extends DialogFragment {
-    public static final String TAG = "DocumentPropertiesFragment";
```

Comment:
Can you leave TAG as it was for now? I don't want to rely on `getSimpleName` without a clear understanding of how it will be impacted by proguard / r8.

Reply:
Sure.